Mathlib/GroupTheory/Perm/Cycle/Basic.lean

```diff
@@ -730,13 +716,78 @@ theorem IsCycle.isCycle_pow_pos_of_lt_prime_order [Finite β] {f : Perm β} (hf
     exact support_pow_le f n
 #align equiv.perm.is_cycle.is_cycle_pow_pos_of_lt_prime_order Equiv.Perm.IsCycle.isCycle_pow_pos_of_lt_prime_order
 
+end Equiv.Perm
+
+namespace Int
+
+open Equiv
+
+theorem _root_.Int.addLeft_one_isCycle : (Equiv.addLeft 1 : Perm ℤ).IsCycle :=
+  ⟨0, one_ne_zero, fun n _ => ⟨n, by simp⟩⟩
+#align int.add_left_one_is_cycle Int.addLeft_one_isCycle
+
+theorem _root_.Int.addRight_one_isCycle : (Equiv.addRight 1 : Perm ℤ).IsCycle :=
+  ⟨0, one_ne_zero, fun n _ => ⟨n, by simp⟩⟩
+#align int.add_right_one_is_cycle Int.addRight_one_isCycle
+
+end Int
+
+section Conjugation
+
+namespace Equiv.Perm
```

Comment:
and here (why did these two lemmas move?)

Reply:
Because I put all the `IsCycle`lemmas together, and then the `IsCycleOn`lemmas. Previously, everything was in an indefinite order.

Reply:
I'm not sure "indefinite order" is very fair. Those lemmas were later because they talk about specific permutations rather than general ones.

If you really want to move those lemmas, you should keep the namespacing untouched and just add `_root_.Int` to their names
```suggestion

theorem _root_.Int.addLeft_one_isCycle : (Equiv.addLeft 1 : Perm ℤ).IsCycle :=
  ⟨0, one_ne_zero, fun n _ => ⟨n, by simp⟩⟩
#align int.add_left_one_is_cycle Int.addLeft_one_isCycle

theorem _root_.Int.addRight_one_isCycle : (Equiv.addRight 1 : Perm ℤ).IsCycle :=
  ⟨0, one_ne_zero, fun n _ => ⟨n, by simp⟩⟩
#align int.add_right_one_is_cycle Int.addRight_one_isCycle

section Conjugation
```

Reply:
By “indefinite”, I meant that the small sections were alternating between stuff about IsCycle and IsCycleOn.  I just put all IsCycle lemmas in one place, and all IsCycleOn lemmas in another place.